### PR TITLE
Fix addition of gco:CharacterString in parent element that was there …

### PR DESCRIPTION
**Pull request by pvalsecc**
…but empty

### CODE REVIEW

**Comment by pvalsecc** [comment]
ping @pmauduit

File: schemas/iso19139/src/main/plugin/iso19139/layout/layout.xsl [hunk 1]
```diff
@@ -408,4 +408,20 @@
       <xsl:with-param name="listOfValues" select="$helper"/>
     </xsl:call-template>
   </xsl:template>
+
+  <!-- Special for textual element addition where the parent was there, but without content -->
+  <xsl:template mode="mode-iso19139" match="gco:CharacterString" priority="0">
+    <xsl:param name="schema" select="$schema" required="no"/>
+    <xsl:param name="labels" select="$labels" required="no"/>
+    <xsl:message>Generic gco <xsl:value-of select="name(.)"/> ==========================</xsl:message>
```
Comment (pmauduit) [inline schemas/iso19139/src/main/plugin/iso19139/layout/layout.xsl]: to be removed ?

**Comment by pmauduit** [comment]
Looks good, but I wonder if this layout.xsl is inherited in the iso19139.che schema

File: schemas/iso19139/src/main/plugin/iso19139/layout/layout.xsl (hunk 1 above)
Comment (pvalsecc) [inline schemas/iso19139/src/main/plugin/iso19139/layout/layout.xsl]: Good catch!

**Comment by pvalsecc** [comment]
It is. Why do you say that? You suggest I move this hack in the CH plugin? I've put it there since it's for the gco namespace... but I'm not 100% against moving it.

**Comment by pmauduit** [comment]
> It is. Why do you say that? You suggest I move this hack in the CH plugin? I've put it there since it's for the gco namespace... but I'm not 100% against moving it.

I was just asking because I did not check the inclusion and since geocat is mainly about iso-che metadatas .. ;-) It sounds more logical to me to leave it this way (gco is related to the regular iso19139 schema, so your current code is ok for me).

**Pull request merged**